smartsim/_core/control/previewrenderer.py

```diff
@@ -90,6 +91,13 @@ def render(
 
     tpl = env.get_template(tpl_path)
 
+    if verbosity_level == Verbosity.INFO:
+        logger.warning(
+            "Only showing user set parameters. Some entity "
+            "fields are truncated. To view truncated fields: use verbosity_"
+            "level 'developer' or 'debug'"
```

Comment:
super nitpick: can you move `fields are` to line 96 so we don't have to rely on concatenation to get `verbosity_level` put back together -> I worry about editing the error message and not paying attention to the concat, and having a nonsense error message.

Reply:
yup